### DISCUSSION TITLE
Introduce aquisition channel breakdown (direct, organic, social...)

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#1828] [Anything Else]
"What I need is a chart that categorizes all the top sources into direct, organic and social so I can get a percentage for each of those 3 things"

**Update:** Channels breakdown is now live in the Top Sources report!

**Comment by cspencer326** [comment]
I would love this too! It would help tremendously with SEO and social media analysis.

**Comment by JohnSharkey** [comment]
Another vote for this one! As it is needed for SEO reporting for search traffic, perhaps a way to filter by a source "type" somehow or by selecting known search engines.

**Comment by metmarkosaric** [comment]
**Update**: It's not exactly this but you can now filter your stats by multiple countries, pages, sources and more, all at once. This is useful to group geographical regions, separate organic search from other sources and segment your audience to your needs. Click on the "Filter" button in your dashboard to start exploring.

**Comment by seamusmh** [comment]
Filtering updates are great. Would love to offer clients that high-level, "at a glance" overview, if it's still on the radar.

**Reply by Jengro777** [reply]
I want to know the top source platforms and methods for my new registered users.

**Comment by metmarkosaric** [comment]
**Update:** Channels breakdown is now live in the Top Sources report!

**Reply by grssnbchr** [reply]
Am I correct to assume that this is also a EE feature / premium and thus not available in CE?
